crates/re_viewer/src/lib.rs

```diff
@@ -154,6 +154,10 @@ pub(crate) fn wgpu_options() -> egui_wgpu::WgpuConfiguration {
             }),
             supported_backends: re_renderer::config::supported_backends(),
             device_descriptor: std::sync::Arc::new(|adapter| re_renderer::config::DeviceCaps::from_adapter(adapter).device_descriptor()),
+
+            present_mode: wgpu::PresentMode::AutoNoVsync, // double-buffered: low-latency, may have tearing
+            // present_mode: wgpu::PresentMode::AutoVsync, // triple-buffered: high-latency, no tearing
```

Comment:
the comments aren't quite accurate. It's very platform dependent what this bottoms out to. There's today no directly control of how many buffers are used.

I picked `AutoVSync` for two reasons:
* on some platforms this bottoms out to adaptive VSync if supported, see https://docs.rs/wgpu/latest/wgpu/enum.PresentMode.html#variant.FifoRelaxed
* don't run unnecessarily hot as `Immediate` does

Wondering if we really want to apply this _always_ 🤔

Reply:
We could make it Mac only.

Do you notice any latency difference on Windows?

Reply:
What do you mean by "run unnecessarily hot"?